django/db/migrations/state.py

```diff
@@ -353,6 +353,10 @@ def __init__(self, app_label, name, fields, options=None, bases=None, managers=N
                     'ModelState.fields cannot refer to a model class - "%s.through" does. '
                     'Use a string reference instead.' % name
                 )
+        # Sanity-check that indexes have their names set.
+        for index in self.options['indexes']:
+            if not index._name:
+                raise ValueError("Indexes passed to ModelState require a name attribute, %r doesn't have one." % index)
```

Comment:
Use:
```python
raise ValueError(
    'Indexes passed to ModelState require a name attribute, '
    '%r doesn't have one.' % index
)
```
While we allow longer lines, we typically break up long strings like this.